core-api/src/main/java/com/optimizely/ab/config/Experiment.java

```diff
@@ -44,6 +45,7 @@
     private final String groupId;
 
     private final List<String> audienceIds;
+    private final Condition audienceConditions;
```

Comment:
I still think generics might be worth considering, since they would provide a more rigid model for how we structure `experiment.audienceConditions` vs. `audience.conditions`. This would also be useful if we introduce other, non-audience-related usages of condition trees in the future (as we have in Web).